src/views/Simulation/utils/pixiUtils.js

```diff
@@ -9,6 +9,8 @@ import { SceneContainer } from './SceneContainer';
 import { PACKAGE_ICON_LINES, GEAR_ICON_LINES, FACTORY_ICON_LINES } from './shapes';
 
 const GRAY_LINE_COLOR = 0xb9bac0;
+const HIGHT_LIGHT_LINE_COLOR = 0xf5f3f3;
```

Comment:
Could we rename the constants?
Maybe we could replace `HIGHT_LIGHT_LINE_COLOR` by simply `LINK_COLOR`, and replace `LOW_LIGHT_LINE_COLOR` by `HIDDEN_LINK_COLOR` or `FILTERED_LINK_COLOR`?